src/elements/Card.vue

```diff
@@ -177,14 +209,14 @@ export default {
     </card>
 
     <!-- Card with sections -->
-    <card id="a" size="full-width">
+    <card id="b" size="full-width">
       <card-media>
         <lux-icon-base width="50" height="50">
           <lux-icon-globe></lux-icon-globe>
         </lux-icon-base>
       </card-media>
       <card-header>
-        <heading level="h2" size="h3">Code4Lib - Trip ID 1234</heading>
+        <heading level="h2" size="h3"><hyperlink href="#">Code4Lib - Trip ID 1234</hyperlink></heading>
```

Comment:
It may be good to explain in the main description why you are wrapping just the `heading` text in the hyperlink. I assume this is what the developer wants screen readers to read out when listing the links on the page?